OpenSim/Simulation/Model/ExpressionBasedBushingForce.cpp

```diff
@@ -177,91 +166,13 @@ void ExpressionBasedBushingForce::extendConnectToModel(Model& aModel)
     setFyExpression(get_Fy_expression());
     setFzExpression(get_Fz_expression());
 
-    string errorMessage;
-    const string& body1Name = get_body_1(); // error if unspecified
-    const string& body2Name = get_body_2();
-
-
-    // Look up the two bodies being connected by bushing by name in the
-    // model. TODO: keep a pointer to them?
-    if (aModel.updBodySet().contains(body1Name)) {
-        // ?Keep a pointer to body1?
-    }
-    else {
-        errorMessage = "Invalid bushing body1 (" + body1Name 
-                        + ") specified in Force " + getName();
-        throw OpenSim::Exception(errorMessage);
-    }
-
-    if (aModel.updBodySet().contains(body2Name)) {
-        // ?Keep a pointer to body2?
-    }
-    else {
-        errorMessage = "Invalid bushing body2 (" + body2Name 
-                        + ") specified in Force " + getName();
-        throw OpenSim::Exception(errorMessage);
+    // fill damping matrix with damping from vector property
+    for (int i = 0; i<3; i++) {
+        _dampingMatrix[i][i] = get_rotational_damping(0)[i];
+        _dampingMatrix[i + 3][i + 3] = get_translational_damping(0)[i];
```

Comment:
It's always bugged me that I didn't spend time designing deflection velocities into the force expressions.  Not in this PR, but soon we should discuss whether or not to treat damping separately as we do no, or incorporate it into the expressions.